index/scorch/snapshot_index_vr.go

```diff
@@ -50,7 +51,19 @@ type IndexSnapshotVectorReader struct {
 	currID        index.IndexInternalID
 	ctx           context.Context
 
-	searchParams json.RawMessage
+	searchParams     json.RawMessage
+	eligibleDocIDs   []index.IndexInternalID
+	requireFiltering bool
+}
+
+func (i *IndexSnapshotVectorReader) EligibleDocIDs() *roaring.Bitmap {
```

Comment:
Don't quite see the need for this to be exported.

Reply:
my bad, fixed it.